tests/credential/conftest.py

```diff
@@ -1,12 +1,9 @@
 import pytest
+from faker import Faker
+from faker.providers import internet, misc
 from selenium.webdriver import Firefox
-from selenium.common.exceptions import WebDriverException
 
-from modules.util import BrowserActions
-from modules.shadow_dom import AboutLogins
-from faker import Faker
-from faker.providers import internet
-from faker.providers import misc
```

Comment:
:+1: Good catch. Did you just 👀 this, or did some linter tooling catch it?

Reply:
that's ruff!